src/app/public/modules/sectioned-form/sectioned-form.component.spec.ts

```diff
@@ -31,14 +31,23 @@ import {
   MockSkyMediaQueryService
 } from '@skyux/core/testing';
 
+function getVisibleTabs(el: any) {
+  return el.querySelectorAll('.sky-sectioned-form-tabs:not(.sky-sectioned-form-tabs-hidden)');
+}
+
 function getVisibleContent(el: any) {
+  return el.querySelectorAll('.sky-sectioned-form-content:not(.sky-sectioned-form-content-hidden)');
+}
+
+function getVisibleTabContent(el: any) {
   return el.querySelectorAll('.sky-vertical-tab-content-pane:not(.sky-vertical-tab-hidden)');
 }
 
 function getActiveSection(el: any) {
   return el.querySelectorAll('sky-sectioned-form-section .sky-vertical-tab-active');
 }
 
+for (let testRunCount = 0; testRunCount < 2; ++testRunCount) {
```

Comment:
While I get what you were going for here I don't know that we really need to rerun every test with the boolean set. Is there a way we could just add a describe block internal to the main `describe` and then add some testing around what changed (with whether or not the contents are destroyed/moved around vs the new behavior where things stay the same).

Same comment for all the spec files. Happy to chat on a call if you'd like about this :)

Reply:
@Blackbaud-TrevorBurch Yeah, I'll take a look at this to.  It's good to know the existing tests don't break with the flag flipped if we make it the default later, but probably not necessary to re-run all of them.  I'll put together a few tests where it tests the changes with the flag flipped to `true`.